### ISSUE TITLE
Add preview feature

### ISSUE THREAD

**Issue by silvenon** [silvenon/vscode-mdx#11]
This is inherently tricky because we have to make some sort of a subset, we have to gracefully degrade what we can't display, like imported React components.

See [vscode-mdx-preview](https://github.com/xyc/vscode-mdx-preview) for inspiration.

**Comment by nkuehn** [comment]
This would be great and an already useful intermediate step would be to allow  triggering a preview in one of the markdown preview extensions (e.g. "Markdown Preview Enhanced").    
Having some sort of preview that ignores the React Parts (or treats them as HTML) is more helpful than none at all and fully evaluating the MDX is not realistically practically in more complex environents as e.g. GastbyJS (experienced with the preview plugin linked above).

**Comment by silvenon** [comment]
Agreed. I currently have zero free time, but in January I could get this working.

**Comment by nkuehn** [comment]
I can't seriously promise time, too - hence no PR but an Issue :-(

**Comment by silvenon** [comment]
Yeah, I immediately deleted that "PR welcome" part from my comment because everyone already knows that 😅

**Comment by silvenon** [comment]
Maybe we can use [react-live](https://github.com/FormidableLabs/react-live), following this example: https://mdxjs.com/guides/live-code
